compiler/src/iree/compiler/Codegen/Common/Passes.cpp

```diff
@@ -17,7 +17,7 @@ void addCommonTargetExecutablePreprocessingPasses(OpPassManager &passManager) {
   nestedModulePM.addPass(createBubbleUpOrdinalOpsPass());
   nestedModulePM.addPass(createBufferizeCopyOnlyDispatchesPass());
   nestedModulePM.addNestedPass<func::FuncOp>(
-      IREE::LinalgExt::createDecomposeSoftmaxPass());
+      IREE::LinalgExt::createDecomposeLinalgLikeOpsPass());
```

Comment:
I would expect that we are dropping this pass completely?

After dropping this, the tile and distrubute should pick up softmax, and the same pass added to LLVMCPU/Passes.cpp / LLVMGPU/Passes.cpp / SPIRV/Passes.cpp should take care of the decomposition.

Reply:
Aah! Good catch! I forgot to remove the pass from here while testing the softmax fusion construct.
Have checked locally after removing it that lit tests still pass - so pushed an updated revision to this PR. :)